qudi/core/gui/gui.py

```diff
@@ -245,6 +289,7 @@ def close_system_tray_icon(self):
         """
         self.hide_system_tray_icon()
         self.system_tray_icon.quitAction.triggered.disconnect()
+        self.system_tray_icon.restartAction.triggered.disconnect()
```

Comment:
Isn't 1h a bit excessive for a sync request timeout? I found little documentation for this parameter but to my understanding this does not mean the maximum time given to execute remote code (e.g. calling a remote function).

Reply:
We had encountered problem for long running jupyter notebooks: they do through the sync error after a while, even though the code was still running in the background. So we solved this by increasing the timeout.

The more elegant solution would be to let rpyc know that something is still running (e.g. by a heartbeat) but I don't know how to do that.